admin-console/pom.xml

```diff
@@ -260,5 +260,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
This may be worht just adding to the parent-pom. I assume we already have this in the engine pom?

Reply:
Yes, the same consideration is valid for the junit dependency.